src/microbe_stage/MicrobeHUD.cs

```diff
@@ -625,6 +625,11 @@ private void UpdateHoverInfo(float delta)
             // Create for each compound the information in GUI
             foreach (var entry in compounds)
             {
+                if (entry.Value < 0.1)
+                {
```

Comment:
I think this would look neater if this didn't use braces to use less vertical space.